python/lsst/analysis/ap/utilities.py

```diff
@@ -0,0 +1,70 @@
+# This file is part of analysis_ap.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["makeSimbadLink"]
+
+from IPython.display import display, Markdown
+import astropy.coordinates as coord
+from astroquery.simbad import Simbad
+import astropy.units as u
+
+
+def makeSimbadLink(source, arcsec_radius=3.0, make_table=False):
```

Comment:
`make_table` appears to be unused?

Reply:
I have removed make_table, as well as the in function print statement since originally it was there to mark if you wanted to print the table or not. If the function is returning the table anyways, then the user can print it then.